### PR TITLE
Test attachment service

### PR DESCRIPTION
**Pull request by EnKarin**
Added a new password for the mail service application

### CODE REVIEW

File: src/main/java/io/github/enkarin/bookcrossing/books/dto/BookModelDto.java [hunk 1]
```diff
@@ -48,7 +48,8 @@ public class BookModelDto {
 
     public Attachment getAttachment() {
         if (attachment != null) {
-            return new Attachment(attachment.getName(), attachment.getData(), attachment.getExpansion());
+            return new Attachment(attachment.getAttachId(), attachment.getBook(), attachment.getData(),
```
Comment (mfvanek) [inline src/main/java/io/github/enkarin/bookcrossing/books/dto/BookModelDto.java:51]: Why return a copy of the attachment here?

Comment (mfvanek) [reply src/main/java/io/github/enkarin/bookcrossing/books/dto/BookModelDto.java:51]: Consider using lombok to make a copy

Comment (EnKarin) [reply src/main/java/io/github/enkarin/bookcrossing/books/dto/BookModelDto.java]: Marked with @Value

File: src/main/java/io/github/enkarin/bookcrossing/books/dto/BookModelDto.java [hunk 2]
```diff
@@ -5,53 +5,39 @@
...
 @SuperBuilder
-@EqualsAndHashCode
+@Value
 @AllArgsConstructor(access = AccessLevel.PRIVATE)
 @Schema(description = "Полные данные книги")
 public class BookModelDto {
 
-    @Getter
     @Schema(description = "Идентификатор", example = "15")
-    private final int bookId;
```
Comment (mfvanek) [inline src/main/java/io/github/enkarin/bookcrossing/books/dto/BookModelDto.java:22]: I'm not happy with such refactoring. Explicit is better than implicit

**Review by mfvanek — APPROVED** [review]

**Pull request merged**